specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -6566,25 +6566,7 @@
       ],
       "x-ms-enum": {
         "name": "StorageAccountTypes",
-        "modelAsString": true,
-        "values": [
-          {
-            "value": "Standard_LRS",
-            "description": "Standard_LRS for StorageAccountType."
-          },
-          {
-            "value": "Premium_LRS",
-            "description": "Premium_LRS for StorageAccountType."
-          },
-          {
-            "value": "StandardSSD_LRS",
-            "description": "StandardSSD_LRS for StorageAccountType."
-          },
-          {
-            "value": "UltraSSD_LRS",
-            "description": "UltraSSD_LRS for StorageAccountType."
-          }
-        ]
```

Comment:
why are we removing the enum values, they provide a reference to possible values supported.

Reply:
@smotwani   Since the enum is set to "modelAsString": true, AutoRest sets its type as string.  And after adding these "values" property, AutoRest throws an exception during SDK generation.  It seems "values" property works only when  "modelAsString" is false.

Reply:
@hyonholee we had this ability to define enum set to "modelAsString": true and still define the possible values allowed/accepted for this enum which get translated to strings in SDK, it is important that we provide the possible string values for this property  as it cannot accept arbitrary string values and consumers of SDK need to aware of these string values, the reason we model enum as string is primarily to ensure backward clients don't when retrieving response for resources created by new clients and we are also able to ensure Get/Put symmetry. 
This limitation where AutoRest throws an exception during SDK generation when specifying values for enum set to modelAsString looks like a recent change and is possibly a regression from previous SDK generation logic.

Reply:
@smotwani  Possible values are still there.  They are defined under "enum".   These "values" property is only for adding description, but somehow AutoRest does not like "values" when "modelAsString": true.

Reply:
@smotwani I think it is OK to delete these values, there is another "enum" field over this deletion to define the possible value.

Reply:
@smotwani I'll not merge until you approve for it.

Reply:
@smotwani  Could you approve it?